### DISCUSSION TITLE
NordVPN - Env Var for Region/Country not working?

### DISCUSSION THREAD

**Discussion by totheroo** [qdm12/gluetun#714] [Q&A]
I may have broken from the pack a bit here; I wasn't feeling confident enough to use compose to build out a stack. That being said, I don't know if that would relate to my issue... which is, no matter what I put into the REGION or COUNTRY environment variables in Portainer and redeploy, it fails to connect to that country and instead seems to pick a server at random.

I have tried US, United States, United_States, "United States", 228 (the country code according to another container's docs)... and so on, I really can't get anything to "stick".

Can anybody help me out? Why isn't Gluetun retaining the env var?

**Comment by qdm12** [comment]
What settings do you have logged out at the beginning of the container?

**Reply by totheroo** [reply]
Here's the log.
[_gluetun_logs.txt](https://github.com/qdm12/gluetun/files/7571881/_gluetun_logs.txt)

Let me know if there's other information you might need from me.

Thanks!

**Reply by qdm12** [reply]
Yes I'm looking for the logs at the start, which seem to not be in the file attached. The region/country filter should be shown there.
